src/Catalog/Helpers/Db2CatalogProjection.cs

```diff
@@ -81,7 +103,7 @@ public PackageDeprecationItem ReadDeprecationInfoFromDataReader(DbDataReader dat
 
                 if (deprecationStatus.HasFlag(deprecationStatusFlag))
                 {
-                    deprecationReasons.Add(deprecationStatusFlag.ToString());
+                    deprecationReasons.Add(deprecationStatusFlag.ToString()); 
```

Comment:
Nit: trailing whitespace

```suggestion
                    deprecationReasons.Add(deprecationStatusFlag.ToString());
```